### PR TITLE
Adding noHistory=true in manifest of login activity

### LINKED ISSUE #1122: On pressing back button from passcode activity login activity is opening.
https://user-images.githubusercontent.com/50478348/103763353-b254dd80-503f-11eb-9f0c-95126534d8f5.mp4

**Summary:**

Pressing the back button opens the login activity.

**Steps to reproduce:**

1. Go to the passcode activity 
2. Hit the back button

**Expected behavior:**

Login activity should only open when the user is logging in for the first time not when we press the back button in the passcode activity.

**Observed behavior:**

On pressing the back button in the passcode Activity login activity is opening.

**Device and Android version:**

Redmi Note 5 Pro
Android 9 Pie
custom ROM

### PR DESCRIPTION
**Pull request by Harshad09**
## Issue Fix
Fixes #1122 

https://user-images.githubusercontent.com/50478348/103763580-0c55a300-5040-11eb-8aea-14b9e985affc.mp4

## Description
In manifest in login activity made noHistory = true.

##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Harshad09** [comment]
@devansh-299  @s-ayush2903 Can you Please Review my PR

File: mifospay/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -35,7 +35,8 @@
         <activity
             android:name=".auth.ui.LoginActivity"
             android:theme="@style/AppTheme"
-            android:windowSoftInputMode="stateVisible" />
+            android:windowSoftInputMode="stateVisible"
+            android:noHistory="true" />
```
Comment (devansh-299) [inline mifospay/src/main/AndroidManifest.xml]: @Harshad09 I don't think it's a good idea to use `noHistory` as we are also creating intent to `MobileVerificationActivity` from this activity from which we may want to return.

Instead, you can probably have a workaround to handle the back press in passcode activity

**Review by devansh-299 — CHANGES_REQUESTED** [review]

**Comment by Harshad09** [comment]
@devansh-299  I have added 
API 16+ use: ```finishAffinity();```  Below API 16 use: ```ActivityCompat.finishAffinity(this);```  in the passcodeActivity

**Comment by devansh-299** [comment]
Closing this pull request (please see [link](https://github.com/openMF/mobile-wallet/issues/1122#issuecomment-1207183580) for reasoning)

**Pull request closed without merging**